lib/gli.rb

```diff
@@ -24,10 +24,20 @@
 
 module GLI
   include GLI::App
+  @@use_hyphenized_form = false
```

Comment:
Yeah, if you have `CommandLineOption` recognize it in its `options` param to the constructor, you should be able to pass it into `Flag.new` and `Switch.new`, which are in `dsl.rb`.  It's not super-clean, but it keeps `CommandLineOption` from having to reach into `GLI` or `App` to figure out what to do.